NOTICE.md

```diff
@@ -22,14 +22,12 @@ https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0 which
 is available at https://www.apache.org/licenses/LICENSE-2.0. This Source Code
 may also be made available under the following Secondary Licenses when the
 conditions for such availability set forth in the Eclipse Public License v. 2.0
-are satisfied: GPL-2.0 with Assembly Exception which is available at
-https://openjdk.java.net/legal/assembly-exception.html, and (secondary) GPL-2.0
-with Classpath-exception-2.0 which is available at GNU General Public License
-v2.0 w/Classpath exception',
-'https://www.gnu.org/software/classpath/license.html.
-
-SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
-OpenJDK-assembly-exception-1.0 OR GPL-2.0-with-classpath-exception
+are satisfied: GPL-2.0 with Classpath-exception-2.0 which is available at
+https://openjdk.java.net/legal/gplv2+ce.html, and GPL-2.0 with Assembly
+Exception which is available at
+https://openjdk.java.net/legal/assembly-exception.html.
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only with Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
```

Comment:
This should use `WITH` (instead of `with`).

The eclipse template should be corrected as well (I see multiple such errors there).

Reply:
The license expression for doxygen is also incorrect.

Reply:
Updated